designer/client/src/stylesheets/editor.scss

```diff
@@ -107,6 +107,8 @@ p code {
   }
 }
 
+$box-border: solid 2px #000;
```

Comment:
Can we keep to `$govuk-*` and `$app-*` suffixes? Cheers @alexluckett 

```suggestion
$app-box-border: solid 2px #000;
```

Reply:
(You'll see the other variables at the top)

Reply:
Done